README.md

```diff
@@ -1 +1,51 @@
-# React-capstone-webapp
\ No newline at end of file
+# Metrics webapp
+> This is the React capstone project at Microverse Program.
+## Built With
+
+- HTML5
+- CSS3
+- JavaScript(ES6)
+- React
+- React Router
+- Redux
+- npm
+- diseases API  (https://disease.sh/)
+
+
+
+
+
+
+## Live Demo
+- at  https://angry-borg-c6fc5f.netlify.app/
+
```

Comment:
- [ ] As it is a requirement for this project, kindly record a video presenting your project and add the link in your readme file, see screenshot below for guidelines. 

![projectPresentation](https://user-images.githubusercontent.com/65852621/147083271-2f1fe795-74e8-40b8-807b-53a6bc9862ed.png)

Reply:
This issue is well resolved! ✔️